client/templates/availability/availability_slot_local.js

```diff
@@ -0,0 +1,11 @@
+import UHR from 'uhr';
```

Comment:
could we also use momentjs for this?

Reply:
`moment` is a Swiss Army Knife for dealing with Date-Time in JavaScript, but still it's a headache to deal with the `Date` object.
`uhr` is specifically design for this purpose only. and has a simple api.
when it comes to performance `uhr` is more efficient than `moment`.
I have setup a benchmark repo here.
https://github.com/distalx/uhr-benchmark

Reply:
I see you have done your research already. lets go with uhr :)